lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -15,6 +15,25 @@ use warnings;
 use File::Find qw(find);
 use URI::file;
 
+# HTML5 namespaces
+our $XPATH = $LaTeXML::Post::Document::XPATH;
```

Comment:
The timing of this is off; it will set `$XPATH` to the current value when `LaTeXML::Post::Manifest::Epub` is loaded, rather than when the document is being packaged. That will probably work for a one-off conversion because LaTeXML has defered but will not (necessarily) be correct in more general (eg. daemon) contexts.

The problem is that we haven't got a clear clean API for managing namespaces so that we can declare these mappings once, and use them consistently throughout conversion & postprocessing (ideally, they should only make it into the otuput document if they're needed).

Short of that, it would be best to move these `registerNS` into one of the `Epub` methods; probably `initialize`.

I'm trying to get some of these PR's moving again. @dginev do you have any comments?

Reply:
I moved most of the `$XPATH` code to `initialize`. Once you reach `initialize`, `XPATH` should already exists, right?